src/Polly.Core/Simmy/Latency/LatencyCompositeStrategyBuilderExtensions.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System.ComponentModel.DataAnnotations;
+using System.Diagnostics.CodeAnalysis;
+
+namespace Polly.Simmy.Latency;
+
+/// <summary>
+/// Extension methods for adding latency to a <see cref="CompositeStrategyBuilderBase"/>.
+/// </summary>
+public static class LatencyCompositeStrategyBuilderExtensions
+{
+    /// <summary>
+    /// Adds a latency chaos strategy to the builder.
+    /// </summary>
+    /// <typeparam name="TBuilder">The builder type.</typeparam>
+    /// <param name="builder">The builder instance.</param>
+    /// <param name="enabled">A value that indicates whether or not the chaos strategy is enabled for a given execution.</param>
+    /// <param name="injectionRate">The injection rate for a given execution, which the value should be between [0, 1].</param>
+    /// <param name="delay">The delay value.</param>
+    /// <returns>The same builder instance.</returns>
+    /// <exception cref="ArgumentNullException">Thrown when <paramref name="builder"/> is <see langword="null"/>.</exception>
+    /// <exception cref="ValidationException">Thrown when the options produced from the arguments are invalid.</exception>
+    public static TBuilder AddLatency<TBuilder>(this TBuilder builder, bool enabled, double injectionRate, TimeSpan delay)
```

Comment:
Do you think we need the convienience extension?
Maybe we can get away with just options-based one? (same as other built-in strategies)

We can still add these later if there is a need for it.

Reply:
I'd really like to keep them since we offered them in the previous version, I think it doesn't hurt